### PR TITLE
added csv support to ip check - AbuseIPDB

### PR DESCRIPTION
**Pull request by moishce**
## Status
Ready

## Related Issues
related: demisto/etc#19784

## Description
Added batch support to IP check commands

## Related PRs
List related PRs against other branches:

branch | PR
 ------ | ------
ipinfo-batch | https://github.com/demisto/content/pull/4699
urlscan.io | https://github.com/demisto/content/pull/4703
circl.lu (CVE Search) | https://github.com/demisto/content/pull/4718
Have I been Pwned | https://github.com/demisto/content/pull/4877

## Required version of Demisto
Any

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Code Review

## Dependencies
- [ ] Impossible Traveler

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/abusedb-batch-support/Integrations/AbuseDB/AbuseDB.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/abusedb-batch-support/Integrations/AbuseDB/CHANGELOG.md)

### CODE REVIEW

File: Integrations/AbuseDB/AbuseDB.py [hunk 1]
```diff
@@ -208,14 +208,19 @@ def createEntry(context_ip, context_ip_generic, human_readable, dbot_scores, tit
...
-        "maxAgeInDays": days
-    }
-    if verbose:
-        params['verbose'] = "verbose"
-    analysis = http_request("GET", url_suffix=CHECK_CMD, params=params).get("data")
-    return analysis_to_entry(analysis, verbose=verbose, threshold=threshold)
+
+    ip_list = argToList(ip)
+    entry_list = []
+    for corrent_ip in ip_list:
```
Comment (Itay4) [inline Integrations/AbuseDB/AbuseDB.py]:
```suggestion
    for current_ip in ip_list:
```

File: Integrations/AbuseDB/AbuseDB.py [hunk 2]
```diff
@@ -208,14 +208,19 @@ def createEntry(context_ip, context_ip_generic, human_readable, dbot_scores, tit
...
+        params = {
+            "ipAddress": corrent_ip,
```
Comment (Itay4) [inline Integrations/AbuseDB/AbuseDB.py]:
```suggestion
            "ipAddress": current_ip,
```

File: TestPlaybooks/playbook-AbuseIPDB-test.yml [hunk 3]
```diff
@@ -177,16 +192,84 @@ tasks:
...
+      script: AbuseIPDB|||ip
+      type: regular
+      iscommand: true
+      brand: AbuseIPDB
+    scriptarguments:
+      days:
+        simple: "10"
+      fullResponse: {}
+      ip:
+        simple: ' 178.62.60.225,8.8.8.8'
```
Comment (Itay4) [inline TestPlaybooks/playbook-AbuseIPDB-test.yml]:
```suggestion
        simple: '178.62.60.225,8.8.8.8'
```

File: Integrations/AbuseDB/AbuseDB.py [hunk 4]
```diff
@@ -208,14 +208,19 @@ def createEntry(context_ip, context_ip_generic, human_readable, dbot_scores, tit
...
-    }
-    if verbose:
-        params['verbose'] = "verbose"
-    analysis = http_request("GET", url_suffix=CHECK_CMD, params=params).get("data")
-    return analysis_to_entry(analysis, verbose=verbose, threshold=threshold)
+
+    ip_list = argToList(ip)
+    entry_list = []
+    for corrent_ip in ip_list:
+        params = {
```
Comment (Itay4) [inline Integrations/AbuseDB/AbuseDB.py]: lets initialize `params` out of the loop, and then assign the ip address to it here

File: Integrations/AbuseDB/AbuseDB.py [hunk 5]
```diff
@@ -208,14 +208,17 @@ def createEntry(context_ip, context_ip_generic, human_readable, dbot_scores, tit
...
-    if verbose:
-        params['verbose'] = "verbose"
-    analysis = http_request("GET", url_suffix=CHECK_CMD, params=params).get("data")
-    return analysis_to_entry(analysis, verbose=verbose, threshold=threshold)
+    params = {}
+    ip_list = argToList(ip)
+    entry_list = []
+    for current_ip in ip_list:
+        params["ipAddress"] = current_ip,
+        params["maxAgeInDays"] = days
```
Comment (Itay4) [inline Integrations/AbuseDB/AbuseDB.py]: lets assign `maxAgeInDays` and `verbose` outside the loop

**Comment by Itay4** [comment]
@kirbles19 @richardbluestone please review the yml and changelog

**Comment by richardbluestone** [comment]
Reviewed both files

**Comment by Itay4** [comment]
@moishce can we merge this?

**Pull request merged**